.github/workflows/e2e.yaml

```diff
@@ -19,12 +19,15 @@ jobs:
         uses: ./.github/actions/setup-go
 
       - name: Install Helm
-        uses: azure/setup-helm@v3.5
+        uses: azure/setup-helm@v3
+        with:
+          version: 'v3.11.1'
+        id: install
```

Comment:
Apparently `helmfile init` installs `helm` too? If that's the case, we can remove this step.

Reply:
Well that's cool